### ISSUE TITLE
Title text get cut off

### ISSUE THREAD

**Issue by FedePorti** [oxyplot/oxyplot#1813]
Hi I was using the oxyplot v2.0.0 and update to v2.1.0.
With the new version I have problems with the text in plot title and in the axes.
This was with the older version:
![image](https://user-images.githubusercontent.com/52248775/139501895-4eb41023-382d-4936-98fe-d398f8128bd8.png)

And this is with the actual version:
![image](https://user-images.githubusercontent.com/52248775/139501981-e3bdc3cd-b026-463c-b5fa-4e339a123d6c.png)

I try using TitleClippingLength = 10 and some title aren't cut off but not all:
![image](https://user-images.githubusercontent.com/52248775/139502273-c9845831-3789-4675-85d5-d673af90752b.png)

Thanks

**Comment by VisualMelon** [comment]
It is possible that the particular provision you are using didn't previously support clipping of text but now does. Which platform are you using?

See the `PlotModel.ClipTitle` and `Axis.ClipTitle` properties, which are better alternatives to setting the `TitleClipLength` to a large value.

In the last image, it looks like are running into issues with the text leaving the plot bounds entirely, which in some vector provisions has 'worked' in the past, but doesn't make a lot of sense. If so, I'm afraid the only workaround would be to render the plot larger and add lots of `PlotModel.Padding` to fill the white-space.

It may make sense for us to make that plot-bound clipping configurable, since plot-area clipping should deal with anything that isn't measurable.

**Comment by FedePorti** [comment]
I working with wpf. 

Yes, ClipTitle does the same that setting  TitleClipLength  to a large value.

But I don't understand why if I put the legends inside the plot the title is okay but if I put it outside the title is cut off. It look like the plot area does not include the legend area.
![image](https://user-images.githubusercontent.com/52248775/139556242-3bdead23-8216-4e1d-a0bf-226fc9a5b5d1.png)
![image](https://user-images.githubusercontent.com/52248775/139556254-26f0c28f-fce3-469f-a98d-bc4e081ad0d7.png)

And here is a question I always have: in the PlotView I setting the MinHeight and MinWidth to a value but I think the plot never grows upper than that value so is like put Height and Width equals to a value.
Is there a way to make the PlotView Heigth and Width to be between a ranges of values?

Thanks.

**Comment by VisualMelon** [comment]
Yes, the title is aligned relative to the plot area. This is something we should consider making configurable.

I'm afraid I don't understand your question about the width/height: can you provide a code sample?

**Comment by VisualMelon** [comment]
It seems it is already configurable: the `PlotModel.TitleHorizontalAlignment` appears to serve the purpose exactly.

**Comment by FedePorti** [comment]
Thanks you @VisualMelon, that work.
![image](https://user-images.githubusercontent.com/52248775/139557811-702160c4-f5d6-4dfe-8535-b0fe38539e72.png)

And my question is for example: in the xaml code when you insert a TextBox you can define the minimum and maximum width and/or heigth. Like you can see that happen in this textbox description in this video.

https://user-images.githubusercontent.com/52248775/139557848-7b497d93-4131-447d-bf32-fed54fd5c32a.mp4

My idea is that the PlotView is between a minimum and maximum value:

![image](https://user-images.githubusercontent.com/52248775/139557889-bb897c7a-f35a-43cf-af52-c5d21175048e.png)

But I suppose is difficult to know what is the best value, maybe it always would be the maximum.

**Comment by VisualMelon** [comment]
Oh with `PlotView`. That behaviour is inherited from Control, and as I understand will essentially be governed by the the `Grid` in your scenario.
